views.py

```diff
@@ -769,5 +776,13 @@ def render_error(error):
         error_code = error.error_code
     else:
         error_code = 500
-    return (flask.render_template('error.html', error=error.message),
+    
+    # Handling Run Time Errors
```

Comment:
Explicitly handling errors that don't have a message (e.g. Value/RunTime Errors)